pyroll/core/roll_pass/hookimpls/roll_pass.py

```diff
@@ -225,4 +225,4 @@ def roll_power_3(self: ThreeRollPass):
 
 @RollPass.entry_angle
 def entry_angle(self: RollPass):
-    return -np.arcsin(self.roll.contact_length / self.roll.working_radius)
+    return np.arcsin(self.roll.contact_length / self.roll.working_radius)
```

Comment:
So the `entry_angle` would be positive since `contact_length` is positive, which does not fit the `neutral_angle`, which is negative.

I suggest adding `entry_point` and using it here, since `contact_length` may include elastic relaxation in future.

Otherwise, just drop this PR.

Reply:
Your right, I will add an entry point. 
Further we should add a sketch of the roll gap to the docs from different perspectives to finally clarify those circumstances.